src/components/Atoms/Shapes/ColoredCircle/ColoredCircle.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react'
+import {
+  StyleSheet,
+  TouchableOpacity,
+} from 'react-native'
+
+import gStyle from '../../../styles'
+
+const size = 160
+
+export default (props) => {
+  const styles = StyleSheet.create({
+    shape: {
+      backgroundColor: props.backgroundColor ? props.backgroundColor : gStyle.colors.primary,
+      alignItems: 'center',
+      justifyContent: 'center',
+      height: size,
+      width: size,
+      borderRadius: 300,
```

Comment:
for a perfect circle, isn't 90 the value to use?

Reply:
As long as the borderRadios >= height and width it'll be a perfect circle, changed the code to reflect this.